plugins/woocommerce/client/admin/client/products/product-variation-page.tsx

```diff
@@ -41,15 +41,19 @@ export default function ProductPage() {
 			render: () => (
 				<>
 					<WooProductMoreMenuItem>
-						{ ( { onClose }: { onClose: () => void } ) => (
-							<>
-								<DeleteVariationMenuItem onClose={ onClose } />
-								<MoreMenuFill
-									productType="product_variation"
-									onClose={ onClose }
-								/>
-							</>
-						) }
+						{
+							( ( { onClose }: { onClose: () => void } ) => (
+								<>
+									<DeleteVariationMenuItem
+										onClose={ onClose }
+									/>
+									<MoreMenuFill
+										productType="product_variation"
+										onClose={ onClose }
+									/>
+								</>
+							) ) as unknown as React.ReactNode
+						}
```

Comment:
Same. :slightly_smiling_face:

Reply:
Fixed in https://github.com/woocommerce/woocommerce/commit/bcd77ab3f81f79d319206f930231bfa7525288f1